Content.Server/Mindshield/MindShieldSystem.cs

```diff
@@ -0,0 +1,81 @@
+using Robust.Server.Player;
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Players;
+using Content.Shared.Implants.Components;
+using Content.Server.Mind;
+using Robust.Shared.Log;
+using Content.Server.Mindshield.Components;
+using Content.Shared.Implants;
+using Content.Shared.Revolutionary;
+using Content.Server.Popups;
+using Robust.Shared.Containers;
+using System.Xml.Linq;
+using Linguini.Bundle.Errors;
+using Content.Server.GameTicking.Rules.Components;
+
+namespace Content.Server.Mindshield;
+/// <summary>
+/// For checking for Mindshield implant at start and giving them the component
+/// </summary>
+public sealed class MindShieldSystem : EntitySystem
```

Comment:
it'd make more sense to just add a revolutionary system on the server and then just put this here, since the two are pretty much entirely linked.d